docs/baseplate/core.rst

```diff
@@ -58,6 +58,43 @@ integrations <integration/index>`.
 .. autoclass:: TraceInfo
    :members: from_upstream
 
+Edge Request Context
+--------------------
+
+If the application is directly receiving requests from a front-end clients or if it
```

Comment:
this whole paragraph is kinda confusing/unnecessary given the next one covers similar stuff. how about giving a bit more concrete of examples of edge request context so people know this section's what they want: "If you want to know more detail about the original request from a user, such as URL or the kind of client, ..."